code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,207 @@
+
+// Defines
+
+#define EXP_TYPE_CREW			"Crew"
+#define EXP_TYPE_COMMAND		"Command"
+#define EXP_TYPE_ENGINEERING	"Engineering"
+#define EXP_TYPE_SECURITY		"Security"
+#define EXP_TYPE_SILICON		"Silicon"
+#define EXP_TYPE_SERVICE		"Service"
+#define EXP_TYPE_MEDICAL		"Medical"
+#define EXP_TYPE_SCIENCE		"Science"
+#define EXP_TYPE_SUPPLY			"Supply"
+#define EXP_TYPE_SPECIAL		"Special"
+#define EXP_TYPE_GHOST			"Ghost"
+#define EXP_TYPE_EXEMPT			"Exempt"
+
+// Globals, Defines, Etc
+
+
+
+
+// Admin Verbs
+
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
+	for(var/client/C in clients)
+		var/list/play_records = params2list(C.prefs.exp)
+		var/gen_exp_time = text2num(play_records[EXP_TYPE_CREW])
+		gen_exp_time = num2text(round(gen_exp_time / 60)) + "h"
+		msg += "<LI> - [key_name_admin(C.mob)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_general() + "</a></LI>"
+	msg += "</UL></BODY></HTML>"
+	src << browse(msg, "window=Player_playtime_check")
+
+
+/client/proc/cmd_show_exp_panel(var/client/C in clients)
+	if(!check_rights(R_ADMIN))
+		return
+	var/body = "<html><head><title>Playtime for [C.key]</title></head><BODY><BR>Playtime:"
+	body += C.get_exp_report()
+	body += "</BODY></HTML>"
+	src << browse(body, "window=playerplaytime;size=550x615")
+
+
+// Procs
+
+
+/datum/job/proc/available_in_playtime(client/C)
+	if(!C)
+		return 0
+	if(!exp_requirements || !exp_type)
+		return 0
+	if(!job_is_xp_locked(src.title))
+		return 0
+	if(check_rights(R_ADMIN, 0, C.mob))
+		return 1
```

Comment:
With the change to what this proc returns, won't this lock all admins out of all jobs?